src/PioneerUtils.jsm

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const { utils: Cu } = Components;
+Cu.import('resource://gre/modules/XPCOMUtils.jsm');
+
+const { TelemetryController } = Cu.import('resource://gre/modules/TelemetryController.jsm', null);
+
+Components.utils.importGlobalProperties(['crypto']);  // Crypto is not available by default
+const { Jose, JoseJWE } = require('jose-jwe-jws/dist/jose-commonjs.js');
```

Comment:
Since we're using webpack > 2, we can use ES6 style import statements, even without Babel.

```js
import { Jose, JoseJWE } from 'jose-jwe-jws/dist/jose-commonjs.js'
```

I think we should.